org.gnome.gThumb.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+
+command: gthumb
+rename-icon: gthumb
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-public-share
```

Comment:
I'm not sure about giving it access to xdg-public-share only. We should at least add `xdg-pictures` here.

Reply:
Is that for folder access? If it's just for files then it should be able to use the portal, no? (note: I have not manually tested that and won't until later today at the earliest)

Reply:
The app is based around folders, rather than individual files, so Portals won't work for the app.
I'm happy to widen the access the app has, although I'd prefer the user to consent more actively into an app accessing data. But I see how the we want to user to have a nice experience with the app just working after installation.